deploy-addresses/op-sepolia.json

```diff
@@ -1,4 +1,3 @@
 {
-  "UniversalRouter": "0x78e7F1502A9e4115dEbd6876E0AC4FaEBDB96880",
```

Comment:
👀 how come you removed this?
This is URV2, it supports v4

Reply:
ohh i thought u said we should remove it https://github.com/Uniswap/universal-router/pull/445#discussion_r1946976326
or should i just change it to URV2?

Reply:
oh yeah youre right lol